docs/1-benchpark-setup.rst

```diff
@@ -0,0 +1,27 @@
+=========
+Benchpark setup
+=========
+
+Python 3.6 and git are required to check out and install Benchpark::
+
+  git clone git@github.com:LLNL/benchpark.git $benchpark  
+  benchpark setup [--spack=spack_dir] [--ramble=ramble_dir]
+
+Benchpark uses the following open source projects for specifying configurations:
+
+* `Ramble <https://github.com/GoogleCloudPlatform/ramble>`_ to specify run configurations
+* `Spack <https://github.com/spack/spack>`_ to specify build configurations
+
+If you already have Spack and/or Ramble on your system and you want to use these
+installs, simply point Benchpark at these installs by passing in their locations.
+
+
+What the ``benchpark setup`` script does for you
+-----------------------------------------
+
+``benchpark setup`` installs Spack and Ramble. If you are running into any issues with either,
+please see the detailed installation instructions in those projects' docs.
+
+Spack: https://spack.readthedocs.io/en/latest/getting_started.html#installation 
+
+Ramble: https://googlecloudplatform.github.io/ramble/getting_started.html#installation 
```

Comment:
In all, `benchpark setup ...` does the following:

* clones spack and ramble into a new directory
* assembles a ramble workspace with a configuration in that directory (using the specified benchmark and system)
* Adds the application/package definitions as Ramble/Spack repositories (to the respective Ramble/Spack instances that were created in the first step)
* prints out instructions on how to build/run the benchmarks for that ramble workspace

this section only mentions the first step. Also, the following line is somewhat vague:

> If you are running into any issues with either,
please see the detailed installation instructions in those projects' docs.

presumably, we (the Benchpark developers) should take charge of any failures in Benchpark to call out properly to Ramble/Spack. Overall, I think bringing them up in this context is not useful and they are sufficiently referenced in `README`.

Reply:
Cleaned up the duplicated text.  This discussion belongs in step 4.